data-es6.js

```diff
@@ -2708,7 +2715,6 @@ exports.tests = [
       res: {
         tr:          true,
         babel:       true,
-        ie11tp:      true,
```

Comment:
Actually, I did some research, and the truth is somewhere in the middle. `/./u` in IETP will wrongly select the first of the surrogate pair, but `/^.$/u` will correctly select the entire character.